scriptharness/__init__.py

```diff
@@ -185,11 +193,11 @@ def get_actions(all_actions):
     """Build a tuple of Action objects for the script.
 
     Args:
-      all_actions (data structure): ordered mapping of action_name:enabled
+      all_actions (Sequence[str, bool]): ordered mapping of action_name:enabled
```

Comment:
This might be right.  all_actions can be a tuple of tuples or list of lists or a dict.  I'm not entirely sure Sequence is accurate, but I'm not sure how to improve it.